typescript/widgets/src/logger.ts

```diff
@@ -1,3 +1,4 @@
 import { rootLogger } from '@hyperlane-xyz/utils';
 
-export const widgetLogger = rootLogger.child({ module: 'widgets' });
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const widgetLogger: any = rootLogger.child({ module: 'widgets' });
```

Comment:
We don't need `any` here. In case of type errors, you can try `rootLogger.child<never>` where never stands for extensions to the set of logging levels.